### PR TITLE
[HK] GeometryPropertyDescriptor no need to use MapEditorPart

### PR DESCRIPTION
**Pull request by fgdrf**
Change-Id: I1de05702e090ea62a4befb79f2d08aace9a8f344
Signed-off-by: Frank Gasdorf <fgdrf@users.sourceforge.net>

### CODE REVIEW

File: plugins/org.locationtech.udig.project.ui/src/org/locationtech/udig/project/ui/internal/properties/GeometryPropertyDescriptor.java [hunk 1]
```diff
@@ -25,14 +25,19 @@
...
 import org.geotools.geometry.jts.JTS;
 import org.locationtech.jts.geom.Envelope;
 import org.locationtech.jts.geom.Geometry;
-import org.locationtech.udig.project.internal.Map;
+import org.locationtech.udig.project.IMap;
+import org.locationtech.udig.project.ILayer;
 import org.locationtech.udig.project.internal.command.navigation.SetViewportBBoxCommand;
-import org.locationtech.udig.project.ui.internal.MapEditorPart;
+import org.locationtech.udig.project.ui.ApplicationGIS;
+import org.locationtech.udig.project.ui.internal.MapPart;
```
Comment (sschulz92) [inline plugins/org.locationtech.udig.project.ui/src/org/locationtech/udig/project/ui/internal/properties/GeometryPropertyDescriptor.java:40]: Is this import used somewhere?

File: plugins/org.locationtech.udig.project.ui/src/org/locationtech/udig/project/ui/internal/properties/GeometryPropertyDescriptor.java [hunk 2]
```diff
@@ -25,14 +25,19 @@
...
+import org.eclipse.ui.IViewReference;
+import org.eclipse.ui.IWorkbenchPage;
+import org.eclipse.ui.IWorkbenchPart;
 import org.eclipse.ui.views.properties.PropertyDescriptor;
 import org.geotools.geometry.jts.JTS;
 import org.locationtech.jts.geom.Envelope;
 import org.locationtech.jts.geom.Geometry;
-import org.locationtech.udig.project.internal.Map;
+import org.locationtech.udig.project.IMap;
+import org.locationtech.udig.project.ILayer;
```
Comment (sschulz92) [inline plugins/org.locationtech.udig.project.ui/src/org/locationtech/udig/project/ui/internal/properties/GeometryPropertyDescriptor.java:37]: Is this import used somewhere?

File: plugins/org.locationtech.udig.project.ui/src/org/locationtech/udig/project/ui/internal/properties/GeometryPropertyDescriptor.java [hunk 3]
```diff
@@ -25,14 +25,19 @@
 import org.eclipse.swt.widgets.Event;
 import org.eclipse.swt.widgets.Label;
 import org.eclipse.swt.widgets.Listener;
-import org.eclipse.ui.PlatformUI;
+import org.eclipse.ui.IViewPart;
+import org.eclipse.ui.IViewReference;
+import org.eclipse.ui.IWorkbenchPage;
+import org.eclipse.ui.IWorkbenchPart;
```
Comment (sschulz92) [inline plugins/org.locationtech.udig.project.ui/src/org/locationtech/udig/project/ui/internal/properties/GeometryPropertyDescriptor.java:31]: Are those imports used somewhere?

**Review by sschulz92 — CHANGES_REQUESTED** [review]

File: plugins/org.locationtech.udig.project.ui/src/org/locationtech/udig/project/ui/internal/properties/GeometryPropertyDescriptor.java (hunk 1 above)
Comment (fgdrf) [reply plugins/org.locationtech.udig.project.ui/src/org/locationtech/udig/project/ui/internal/properties/GeometryPropertyDescriptor.java]: nice catch! re-organized imports ;)

**Comment by sschulz92** [comment]
Looks good to me 👍

**Review by sschulz92 — APPROVED** [review]
👍

**Pull request merged**